step-controller/step-controller-server/src/main/java/step/controller/services/entities/AbstractEntityServices.java

```diff
@@ -197,4 +198,36 @@ public History(String id, long updateTime) {
     public T restoreVersion(@PathParam("id") String id, @PathParam("versionId") String versionId) {
         return accessor.restoreVersion(new ObjectId(id), new ObjectId(versionId));
     }
+
+    @Operation(operationId = "is{Entity}Locked", description = "Get entity locking state")
+    @GET
+    @Path("/{id}/locked")
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Not sure if the boolean result of the method matches with this. This might create errors in client generators